core/platform/models_test.py

```diff
@@ -406,4 +406,4 @@ def test_import_models_not_implemented_has_not_implemented_error(self):
             re.escape(
                 'import_models() method is not overwritten in '
                 'derived classes')):
-            models.Platform().import_models()
+            models.Platform().import_models([''])
```

Comment:
Nit: it would be slightly better to use an actual model name here, so that it's clear that the error isn't due to what is obviously an invalid string.

Reply:
Done